MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -57,44 +58,27 @@
 /*! The redirect URI of the application */
 @property (readonly, nonnull) MSALRedirectUri *redirectUri;
 
-/*! When checking an access token for expiration we check if time to expiration
- is less than this value (in seconds) before making the request. The goal is to
- refresh the token ahead of its expiration and also not to return a token that is
- about to expire. */
-@property NSUInteger expirationBuffer;
-
-/*!
- List of additional ESTS features that client handles.
- */
-@property (nullable) NSArray<NSString *> *clientCapabilities;
-
-/*!
-    Used to specify query parameters that must be passed to both the authorize and token endpoints
-    to target MSAL at a specific test slice & flight. These apply to all requests made by an application.
- */
-@property (nullable) NSDictionary<NSString *, NSString *> *sliceParameters;
```

Comment:
Should we keep slice parameters? It has been there from the beginning so perhaps someone is using it?

Reply:
This was a part of deprecation effort, but without it, let's bring it back.